<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GkukanMusiumdbDockWidgetBase</class>
 <widget class="QDockWidget" name="GkukanMusiumdbDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/MGMusium.png</normaloff>icons/MGMusium.png</iconset>
  </property>
  <property name="locale">
   <locale language="Japanese" country="Japan"/>
  </property>
  <property name="windowTitle">
   <string>gkukanmusiumdb</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tblPhotos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>3</number>
      </property>
      <property name="columnCount">
       <number>20</number>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/plugins/GkukanMusiumdb/icons/MGMusium.png&quot; height=&quot;16&quot;/&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;松江Ｇ空間ミュージアム&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="FilterNoEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>位置未指定のみ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReloadphoto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>再読込み</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadMorePhoto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>さらに読込む</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearSelection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>選択クリア</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetToPoint">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>位置指定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetLandmark">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ランドマークへ追加</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.py"/>
 </resources>
 <connections>
  <connection>
   <sender>FilterNoEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>FilterNoEdit</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetLandmark</sender>
   <signal>clicked()</signal>
   <receiver>btnSetLandmark</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblPhotos</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>tblPhotos</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearSelection</sender>
   <signal>clicked()</signal>
   <receiver>btnClearSelection</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetToPoint</sender>
   <signal>clicked()</signal>
   <receiver>btnSetToPoint</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblPhotos</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>tblPhotos</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
